lib/castle/commands/list/create.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module Castle
+  module Commands
+    module List
+      class Create
+        class << self
+          # @param options [Hash]
```

Comment:
[note] could be useful to explain option expectations for these commands, but would could get out of date if api changes

Reply:
fwiw this is my vote if you see value in inline options documentation

Reply:
Let's keep it simple here. As you correctly pointed out, API could change. Official API reference is a better source of truth in that case.